.github/scripts/build_without_nix_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+nix develop -c "env" | grep CMAKE_PREFIX_PATH >cmake_prefix
```

Comment:
first command in file `build_without_nix_env` is `nix_develop`)
maybe just rename script to `pure_cmake_build`

Reply:
Done